src/ai/backend/manager/models/keypair.py

```diff
@@ -228,6 +234,20 @@ async def resolve_num_queries(self, info: graphene.ResolveInfo) -> int:
             return n
         return 0
 
+    async def resolve_rolling_count(self, info: graphene.ResolveInfo) -> int:
+        ctx: GraphQueryContext = info.context
+        redis_rlim = redis_helper.get_redis_object(
+            ctx.shared_config.data["redis"], name="ratelimit", db=REDIS_RLIM_DB
+        )
+        ret = await redis_helper.execute_script(
+            redis_rlim,
+            "rollingcount",
+            _rolling_count_script,
+            [self.access_key],
+            [],
+        )
```

Comment:
Since the script now contains only a single command ZCARD, there is no need to use the complex `execute_script()` API. :wink: